src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1150,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
This method does not have the expected behaviour. It should replace if the prefix exists at the beginning of the method. 

You have not updated the header comments to reflect the new change. 

When you modify the behavior, remember to update documentation and tests as well.